specification/compute/resource-manager/Microsoft.Compute/galleryRP/stable/2022-03-03/gallery.json

```diff
@@ -2606,7 +2774,7 @@
     "GalleryImageVersionStorageProfile": {
       "properties": {
         "source": {
-          "$ref": "#/definitions/GalleryArtifactVersionSource"
+          "$ref": "#/definitions/GalleryArtifactVersionFullSource"
```

Comment:
This is also a breaking change...
We need to find a solution for this.

Reply:
See comment on other thread